packages/webapp/src/routes/land/client-individual-organisation.js

```diff
@@ -0,0 +1,36 @@
+import constants from '../../utils/constants.js'
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(constants.views.CLIENT_INDIVIDUAL_ORGANISATION)
+  },
+  post: async (request, h) => {
+    const { landownerType } = request.payload
+
+    if (!landownerType) {
+      return h.view(constants.views.CLIENT_INDIVIDUAL_ORGANISATION, {
+        err: [{
+          text: 'Select if your client is an individual or organisation',
+          href: '#landownerType'
+        }]
+      })
+    }
+    request.yar.set(constants.redisKeys.CLIENTS_ORGANISATION_NAME, landownerType)

```

Comment:
If this was my request it needs to be `constants.redisKeys.CLIENT_INDIVIDUAL_ORGANISATION`

Reply:
CLIENTS_ORGANISATION_NAME is definitely a mistype i'm sure

Reply:
Oh and obviously CLIENT_INDIVIDUAL_ORGANISATION will need adding to the redisKeys in constants too :)

Reply:
you are right, its a mistype